config/locales/en.yml

```diff
@@ -693,7 +694,7 @@ en:
       blog_link_text: "Project blog"
       changelog: "%{changelog_link} – a record of diaspora*’s development, including features not yet released in the stable version"
       changelog_link_text: "The changelog"
-      loomio: "the online collaborative tool we use to discuss and vote on proposals to improve diaspora*"
+      discourse: "the platform we use to discuss about ideas and improvements for diaspora*"
```

Comment:
Just one tiny point: 'to discuss about ideas ...' would be better as 'to discuss ideas ...' - 'discuss about' just doesn't sound right.